.github/workflows/ci-lint.yaml

```diff
@@ -1,4 +1,4 @@
-name: PR - Linting
+name: Lint backend
```

Comment:
The renaming you see in this file (of the workflow and of the job below) will hopefully make things more readable for the PR checks